apprentice-react-o-shop/src/components/Main/Main.js

```diff
@@ -1,11 +1,29 @@
 import React from 'react';
+import { Route, Switch } from 'react-router';
 import './Main.scss'
+import AdminOrders from '../../pages/AdminOrders'
+import AdminProducts from '../../pages/AdminProducts'
+import CheckOut from '../../pages/CheckOut'
+import Home from '../../pages/Home'
+import Orders from '../../pages/Orders'
+import OrderSuccess from '../../pages/OrderSuccess'
+import Products from '../../pages/Products'
+import ShoppingCart from '../../pages/ShoppingCart'
 
 function Main() {
     return (
-        <main>
-            <h2>This is the Main Component</h2>
-        </main>
+        <div className="main">
+            <Switch>
+                <Route exact path="/" component={ Home }></Route>
```

Comment:
since we do not put anything between the opening and closing tags of Route you can make it a self closing tag